Sources/Views/Collections/CollectionSection.swift

```diff
@@ -197,21 +199,20 @@ private func relatedItem(_ relatedItem: Episode.Collection.Section.Related) -> N
           Class.flex.items.center,
           Class.grid.row,
           Class.margin([
-            .desktop: [.top: 2, .bottom: 3],
-            .mobile: [.top: 1, .bottom: 2],
+            .desktop: [.top: 2],
+            .mobile: [.top: 1],
           ]),
           Class.padding([.mobile: [.leftRight: 2]]),
           Class.pf.collections.hoverBackground,
           Class.pf.collections.hoverLink,
           Class.pf.colors.border.gray800,
           Class.pf.colors.bg.gray900,
         ]),
-        .href(url(to: .episode(.show(.right(episode.id))))),
+        .href(url(to: .episode(.show(.left(episode.slug))))),
```

Comment:
I think our current route doesn't even work with `.right(Episode.Id)`s. should probably just remove that version of the route, doesn't seem we even use it